### ISSUE TITLE
Elden Ring save file conflicts with seamless coop

### ISSUE THREAD

**Issue by melak47** [soulsmods/ModEngine2#54]
When I use this mod engine 2 together with the seamless coop mod (https://www.nexusmods.com/eldenring/mods/510),
sometimes the game loads the vanilla saves, and sometimes the coop saves.
Seamless coop changes the savegame file names the game saves to (keeping the coop and vanilla saves separate), maybe there's a race condition between that (loaded via mod loader through dinput8.dll) and mod engine 2?

**Comment by Pav-Osmolski** [comment]
I was able to resolve this issue by creating a folder in `mods\elden_ring_seamless_coop\` and setting the `load.txt` to have a priority of 0.

Now it always loads the .co2 save file without any issues.

**Comment by PhantomFlayer** [comment]
FYI, you should set the load priority to "1" as opposed to "0". Setting it to "0" forces the .dll to load without any delay, which can cause issues.

**Comment by Pav-Osmolski** [comment]
> FYI, you should set the load priority to "1" as opposed to "0". Setting it to "0" forces the .dll to load without any delay, which can cause issues.

I do not believe that 0 causes an issue with the Seamless Coop mod. I've had it set at 0 the entire time, across three separate computers (we play it LAN) and never encountered a single issue re: initialising.

**Comment by melak47** [comment]
I've tried a bunch of these things, and changing the load order in mod loader doesn't seem to fix it for me.
It still happens even if seamless coop is the only mod loaded.

For now I've just moved my single player saves elsewhere,
and made the *.sl2 files symlinks pointing to the *.co files...

**Comment by mike3056** [comment]
idk how to symlinks and the 0 method did not work rip

**Comment by SoulStorm781** [comment]
I've completed the game once, and now I've started NG+1, but it seems too easy. I want a NG+7 save file for seamless co-op. If anyone can provide it or help me find it, that would be appreciated.
